web-registry/src/pages/Home/Home.tsx

```diff
@@ -59,9 +56,14 @@ const Home: React.FC = () => {
   return (
     <Box sx={{ backgroundColor: 'primary.main' }}>
       <BackgroundImgSection
-        img={heroSection?.background?.image?.asset?.url || ''}
-        linearGradient="linear-gradient(180deg, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 100%);"
+        img={
+          'https://cdn.sanity.io/images/jm12rn9t/production/ae7630b9aa71e3d701b692ed576ba0fbd4c0433a-1440x760.jpg?w=2000&fit=max&auto=format' ||
```

Comment:
The sanity URL is temporary to be able to QA the new design on the preview env. Publishing on sanity will apparently publish in production too so we'll need to merge this PR before publishing sanity new changes.